src/modules/cpu/kernel/color_temperature.hpp

```diff
@@ -0,0 +1,984 @@
+#include "rppdefs.h"
+#include "rpp_cpu_simd.hpp"
+#include "rpp_cpu_common.hpp"
+
+RppStatus color_temperature_u8_u8_host_tensor(Rpp8u *srcPtr,
+                                              RpptDescPtr srcDescPtr,
+                                              Rpp8u *dstPtr,
+                                              RpptDescPtr dstDescPtr,
+                                              Rpp8s *adjustmentValueTensor,
+                                              RpptROIPtr roiTensorPtrSrc,
+                                              RpptRoiType roiType,
+                                              RppLayoutParams layoutParams)
+{
+    RpptROI roiDefault = {0, 0, (Rpp32s)srcDescPtr->w, (Rpp32s)srcDescPtr->h};
+
+    omp_set_dynamic(0);
+#pragma omp parallel for num_threads(dstDescPtr->n)
+    for(int batchCount = 0; batchCount < dstDescPtr->n; batchCount++)
+    {
+        RpptROI roi;
+        RpptROIPtr roiPtrInput = &roiTensorPtrSrc[batchCount];
+        compute_roi_validation_host(roiPtrInput, &roi, &roiDefault, roiType);
+
+        Rpp32f adjustmentValue = adjustmentValueTensor[batchCount];
+
+        Rpp8u *srcPtrImage, *dstPtrImage;
+        srcPtrImage = srcPtr + batchCount * srcDescPtr->strides.nStride;
+        dstPtrImage = dstPtr + batchCount * dstDescPtr->strides.nStride;
+
+        Rpp32u bufferLength = roi.xywhROI.roiWidth * layoutParams.bufferMultiplier;
+
+        __m256 pAdj = _mm256_set1_ps(adjustmentValue);
```

Comment:
please combine line #24 into this and remove line #24

Reply:
We will need the adjustmentValue parameter in 8 other locations below though, which are all inside loops within one large omp loop. So it might be better to create a local variable for that?

Reply:
Sure Rajy I will modify this.